app/styles/networkvis.less

```diff
@@ -42,19 +44,28 @@ svg {
 
   }
 
+  .link-selected {
```

Comment:
Awesome improvement to move the styling to css, I've been wondering how much can go in css vs how much if anything must be in the javascript for svg.